src/Controller/AdminController.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace App\Controller;
+
+use App\Model\AdminRoomManager;
+use App\Services\AddPictures;
+use App\Services\CleanForm;
+
+class AdminController extends AbstractController
+{
+
+    public function addroom()
```

Comment:
Method names MUST be declared in camelCase()
Transforme le nom de ta fonction en camelcase :D

Reply:
OK, corrigé